src/test/java/mate/academy/validation/EmailValidatorTest.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.validation;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class EmailValidatorTest {
+    private static final String INVALID_EMAIL_ONE_DOMAIN = "test@a";
+    private static final String INVALID_EMAIL_UNFINISHED = "test@a.";
+    private static final String VALID_EMAIL = "test@gmail.com";
+    private static EmailValidator validator;
+
+    @BeforeAll
+    static void beforeAll() {
+        validator = new EmailValidator();
+    }
+
+    @Test
+    void isValid_validNullValue_notOk() {
```

Comment:
"notOK" is the case when we are expected exception, so in this case, better name method with the ending ok, check it everywhere